frontend/public/components/storage/create-pvc.tsx

```diff
@@ -156,13 +131,8 @@ export const CreatePVCForm: React.FC<CreatePVCFormProps> = (props) => {
     setPvcName(event.currentTarget.value.trim());
   };
 
-  const handleAccessMode: React.ReactEventHandler<HTMLInputElement> = (event) => {
-    setAccessMode(event.currentTarget.value);
-  };
-
-  const handleVolumeMode: React.ReactEventHandler<HTMLInputElement> = (event) => {
-    setVolumeMode(event.currentTarget.value);
-  };
+  const handleAccessMode = (value: string) => setAccessMode(value);
+  const handleVolumeMode = (value: string) => setVolumeMode(value);
```

Comment:
These helpers are unnecessary as they have the same signature as the functions they wrap.